website/content/docs/agent/options.mdx

```diff
@@ -670,8 +685,16 @@ Valid time units are 'ns', 'us' (or 'µs'), 'ms', 's', 'm', 'h'."
       register as in order to set any of the node-level information in the
       catalog such as metadata, or the node's tagged addresses.
 
-    - `agent_master` ((#acl_tokens_agent_master)) - Used to access [agent endpoints](/api/agent)
-      that require agent read or write privileges, or node read privileges, even if Consul servers aren't present to validate any tokens. This should only be used by operators during outages, regular ACL tokens should normally be used by applications.
+    - `agent_recovery` ((#acl_tokens_agent_recovery)) - Used to access [agent endpoints](/api/agent)
+      that require agent read or write privileges, or node read privileges, even if Consul servers
+      aren't present to validate any tokens. This should only be used by operators during outages,
+      regular ACL tokens should normally be used by applications.
+
+    - `agent_master` ((#acl_tokens_agent_master)) **Deprecated in Consul 1.11. Use the
```

Comment:
Same comment as above (with master -> initial_management). Can we just reference rather than copy the docs text?